CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# dbt_netsuite v0.10.0
+[PR #84](https://github.com/fivetran/dbt_netsuite/pull/84)includes the following updates:
+## 🚨 Breaking Changes 🚨
+- For **Netsuite2**: updated the following models to reference `account_type_id` instead of `type_name`:
+  - int_netsuite2__tran_with_converted_amounts
+  - netsuite2__balance_sheet
+  - netsuite2__income_statement
+  - netsuite2__transaction_details
+- The above change was implemented for Netsuite2 because `type_name` was previously utilized to categorize records, which was causing issues for users that customized the `type_name` values. Utilizing the unique identifier `account_type_id` instead produces more accurate results in the final models. Below is a list of the account type name and type id pairings used:
```

Comment:
Can you add the link here for where the pairings came from.

Reply:
added